src/schemas/json/web-manifest-app-info.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "JSON schema for Web Application manifest files with app information extensions",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "type": "object",
+
+  "properties": {
+    "categories": {
+      "description": "Describes the expected application categories to which the web application belongs.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "description": {
+      "description": "Description of the purpose of the web application",
+      "type": "string"
+    },
+    "iarc_rating_id": {
+      "description": "Represents an ID value of the IARC rating of the web application. It is intended to be used to determine which ages the web application is appropriate for.",
+      "type": "string"
+    },
+    "screenshots": {
+      "description": "The screenshots member is an array of image objects represent the web application in common usage scenarios.",
+      "type": "array",
+      "items": {
+        "$ref": "web-manifest.json#/definitions/manifest_image_resource"
```

Comment:
Have you tested this relative URL works in VS/VSCode or should it be an absolute URL instead?

Reply:
I'm using absolute URLs now, hope that's okay. I haven't tested it in an IDE though, how can I do that?